plugins/woocommerce/src/Blocks/BlockTypes/ProductFilters.php

```diff
@@ -100,12 +100,17 @@ function ( $carry, $parsed_block ) use ( $block_context ) {
 			'data-wp-interactive'              => $this->get_full_block_name(),
 			'data-wp-watch--scrolling'         => 'callbacks.scrollLimit',
 			'data-wp-on--keyup'                => 'actions.closeOverlayOnEscape',
-			'data-wp-router-region'            => $this->generate_navigation_id( $block ),
 			'data-wp-context'                  => wp_json_encode( $interactivity_context, JSON_HEX_TAG | JSON_HEX_APOS | JSON_HEX_QUOT | JSON_HEX_AMP ),
 			'data-wp-class--is-overlay-opened' => 'context.isOverlayOpened',
 			'style'                            => $styles,
 		);
 
+		// phpcs:ignore
```

Comment:
I can see that you put the ignore here to avoid the warning with todo below, but I think the warning is actually needed to remind us about the todo. Does CI complaint about that or do you have any other reason?

Reply:
Yeah I am getting warning in the IDE...Not sure in CI though.

Reply:
Then I think we should try removing that ignore to use the warning for our benefit. That's my only remaining recommendation. We have multiple todos in our codebase, none of them is ignored.

Reply:
Done here [cdbdf62](https://github.com/woocommerce/woocommerce/pull/56038/commits/cdbdf62699b1371d3dc4f133bc03e02c806ed570)